src/Symfony/Component/Translation/Loader/XliffFileLoader.php

```diff
@@ -109,77 +109,23 @@ private function utf8ToCharset($content, $encoding = null)
     }
 
     /**
-     * Validates and parses the given file into a SimpleXMLElement.
+     * Validates and parses the given file into an array that contains a SimpleXml instance and the encoding.
      *
      * @param string $file
      *
-     * @throws \RuntimeException
-     *
-     * @return \SimpleXMLElement
+     * @return array
      *
+     * @throws \RuntimeException
      * @throws InvalidResourceException
      */
     private function parseFile($file)
     {
         try {
```

Comment:
you can remove try/catch block here as it's already covered in [XliffUtils](https://github.com/marcosdsanchez/symfony/blob/xliff-utils/src/Symfony/Component/Translation/Util/XliffUtils.php#L66-L70)